doc/schemas/api-status.json

```diff
@@ -13,6 +13,14 @@
       "format": "uri",
       "description": "URL of the GitGuardian instance"
     },
+    "instance_source": {
+      "type": "string",
+      "description": "Source the instance was read from"
+    },
+    "api_key_source": {
+      "type": "string",
+      "description": "Source the API key was read from"
+    },
```

Comment:
The `type` for these fields should be `enum` and the values should be the internal enum values, not their user-friendly texts (so `CMD_OPTION`, `ENV_VAR`...). This way we can reword the user-friendly texts without breaking anything.

Reply:
Yes, the test would need to check that we get human-friendly names in text mode, but machine-friendly enum values in JSON mode.